contracts/src/v0.8/keystone/CapabilityRegistry.sol

```diff
@@ -143,15 +160,31 @@ contract CapabilityRegistry is OwnerIsCreator, TypeAndVersionInterface {
     uint32 id;
     /// @notice The number of times the DON was configured
     uint32 configCount;
+    /// @notice The f value for the DON.  This is the number of faulty nodes
+    /// that the DON can tolerate. This can be different from the f value of
+    /// the OCR instances that capabilities spawn.
+    uint32 f;
     /// @notice True if the DON is public.  A public DON means that it accepts
     /// external capability requests
     bool isPublic;
+    /// @notice True if the DON accepts Workflows.
+    bool acceptsWorkflows;
     /// @notice List of member node P2P Ids
     bytes32[] nodeP2PIds;
     /// @notice List of capability configurations
     CapabilityConfiguration[] capabilityConfigurations;
   }
 
+  /// @notice DONParams is a struct that holds the parameters for a DON.
+  /// @dev This is needed to avoid "stack too deep" errors in _setDONConfig.
+  struct DONParams {
+    uint32 id;
```

Comment:
Add natspec comments for the params

Reply:
Will do in a follow-up to avoid losing approvals on this PR